go/kbfs/libkbfs/settings_db.go

```diff
@@ -131,32 +136,38 @@ func (db *SettingsDB) Settings(ctx context.Context) (keybase1.FSSettings, error)
 	var notificationThreshold int64
 	notificationThresholdBytes, err :=
 		db.Get(getSettingsDbKey(uid, spaceAvailableNotificationThresholdKey), nil)
-	if err != nil {
-		// If we have an error we just pretend there's an empty setting.
-		return keybase1.FSSettings{
-			SpaceAvailableNotificationThreshold: 0,
-			SfmiBannerDismissed:                 false,
-		}, nil
-	}
-	notificationThreshold, err =
-		strconv.ParseInt(string(notificationThresholdBytes), 10, 64)
-	if err != nil {
+	switch errors.Cause(err) {
+	case leveldb.ErrNotFound:
+		db.logger.CDebugf(ctx,
+			"notificationThreshold not set; using default value")
+	case nil:
+		notificationThreshold, err =
+			strconv.ParseInt(string(notificationThresholdBytes), 10, 64)
+		if err != nil {
+			return keybase1.FSSettings{}, err
+		}
+	default:
+		db.logger.CWarningf(ctx,
+			"reading notificationThreshold from leveldb error: %v", err)
```

Comment:
Super minor: it's nice if errors are printed with `%+v` so we can see the full stack for errors with stacktraces. (Here and below.)